topics/valkey-clients.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Valkey Clients"
+linkTitle: "Valkey Clients"
```

Comment:
In the list of pages, we don't want all of them to start with Valkey.

I believe "linkTitle" is used in menues (@stockholmux correct me if I'm wrong!), where we see all of these together, and "title" is the title of the page.

For linkTitle, we don't want to repeat Valkey everywhere. We already have this for half of the pages and it looks like crap. See "Alphabetical index" on the page https://valkey.io/topics/ and scroll down.

Reply:
Clarification: The page itself doesn't look like crap. It's looks pretty IMO. But it's not nice that half of them start with "Valkey".

Reply:
Good to know. So we can delete linkTitle to avoid confusion.

Should we skip "Valkey" in the page titles then, or use it in all page titles?

Reply:
@zuiderkwast (Slightly off topic)
Yeah, so it renders as `<title>Valkey Documentation · Valkey persistence</title>` which is a little silly.

Do you have a need for it to be "Valkey <topic>" vs just "<topic>"? If so, it's pretty trivial to just remove "Valkey" from the `title` string in the template rendering for things like the alphabetical index and document title. Feel free to put in an issue; I'm chewing through a backlog slowly.

I also have a PR in review that allows for conditional overrides of front matter directly into Zola so in the few edge cases where we might want to preserve 'Valkey' we can.

Reply:
Yes, I'd like to remove Valkey from all the titles in markdown, so it renders as "Valkey Documentation · Persistence" instead.

The alphabetical index looks especially silly when all or most of the entries start with Valkey.

Reply:
@viktor See PR that fixes this: 
https://github.com/valkey-io/valkey-io.github.io/pull/169

Reply:
deleted the `linkTitle`